<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="XSE6CLNM675W4JT" ignored="0" name="Test" tests="6" failures="3" timestamp="24 一月 2018 11:55:06 GMT" time="13.827" errors="0">
  <testcase name="TestngDemo" time="6.612" classname="demo20180107.TestngExe">
    <skipped/>
  </testcase> <!-- TestngDemo -->
  <testcase name="TestngDemo" time="0.099" classname="demo20180107.TestngExe">
    <skipped/>
  </testcase> <!-- TestngDemo -->
  <testcase name="TestngDemo" time="0.078" classname="demo20180107.TestngExe">
    <skipped/>
  </testcase> <!-- TestngDemo -->
  <testcase name="TestngDemo" time="0.037" classname="demo20180107.TestngExe">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]at demo20180107.TestngExe.TestngDemo(TestngExe.java:60)... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TestngDemo -->
  <testcase name="TestngDemo" time="0.458" classname="demo20180107.TestngExe">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]at demo20180107.TestngExe.TestngDemo(TestngExe.java:60)... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TestngDemo -->
  <testcase name="TestngDemo" time="0.066" classname="demo20180107.TestngExe">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]at demo20180107.TestngExe.TestngDemo(TestngExe.java:60)... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TestngDemo -->
</testsuite> <!-- Test -->
